pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -186,22 +166,28 @@ func resolveTask(ctx context.Context, resolver remote.Resolver, name string, kin
 // An error is returned if the given object is not a Task nor a ClusterTask
 // or if there is an error validating or upgrading an older TaskObject into
 // its v1beta1 equivalent.
+// v1beta1 task will be verified by trusted resources if the feature is enabled
 // TODO(#5541): convert v1beta1 obj to v1 once we use v1 as the stored version
-func readRuntimeObjectAsTask(ctx context.Context, obj runtime.Object) (*v1beta1.Task, error) {
+func readRuntimeObjectAsTask(ctx context.Context, obj runtime.Object, k8s kubernetes.Interface, refSource *v1beta1.RefSource, verificationPolicies []*v1alpha1.VerificationPolicy) (*v1beta1.Task, error) {
 	switch obj := obj.(type) {
 	case *v1beta1.Task:
+		err := trustedresources.VerifyTask(ctx, obj, k8s, refSource, verificationPolicies)
```

Comment:
can you also add a comment here please?

Reply:
Oh sorry 😞 , I thought I added it but I should have double-checked